cypress/e2e/studentsBasicInfo.cy.ts

```diff
@@ -0,0 +1,125 @@
+    
+    const placeOfBirthFieldID = "#\\#\\/properties\\/birthplace2-input";
+    const placeOfBirthFieldErrorID = "#\\#\\/properties\\/birthplace2> :nth-child(3)";
+    
+    const dateOfBirthInputID = "#\\#\\/properties\\/birthdate2-input";
+    const dateOfBirthID = ":nth-child(4) > .MuiGrid-container > :nth-child(2) > .MuiFormControl-root";
+    
+    const nationalityFieldID = "#\\#\\/properties\\/nationality2-input";
+    const nationalityFieldErrorID = "#\\#\\/properties\\/nationality2> :nth-child(3)";
+    
+    const firstLanguageFieldID = "#\\#\\/properties\\/first_language2-input";
+    const firstLanguageFieldErrorID = "#\\#\\/properties\\/first_language2> :nth-child(3)";
+    
+    const neighborhoodFieldID = "#\\#\\/properties\\/neighborhood2-input";
+    const neighborhoodFieldErrorID = "#\\#\\/properties\\/neighborhood2> :nth-child(3)";
+    
+    const addressFieldID = "#\\#\\/properties\\/address2-input";
+    const addressFieldErrorID = "#\\#\\/properties\\/address2> :nth-child(3)";
+    
+    const medicalAssuranceFieldID = "#\\#\\/properties\\/medical_assurance2-input";
+    const medicalAssuranceFieldErrorID = "#\\#\\/properties\\/medical_assurance2> :nth-child(3)";
+    
+    const emergencyFieldID = "#\\#\\/properties\\/emergency2-input";
+    const emergencyFieldErrorID = "#\\#\\/properties\\/emergency2> :nth-child(3)";
+    
+    const vaccineExpirationInputID = "#\\#\\/properties\\/vaccine_expiration2-input";
+    const vaccineExpirationID = ":nth-child(6) > .MuiGrid-container > :nth-child(3) > .MuiFormControl-root";
+
+    const createButtonID = ".MuiBox-root > .MuiButtonBase-root"
+
+    
+    beforeEach(() => {
+        cy.login();
+    
+        cy.wait(300);
+    
+        cy.visit("/students");
+    });
+
+    it("does not show error messages before inputting info on any field", () => {
+		cy.get(basicInfoButtonID).click();
+
+		cy.get(nameFieldErrorID).should("not.exist");
+		cy.get(surnameFieldErrorID).should("not.exist");
+		cy.get(CIFieldErrorID).should("not.exist");
+		cy.get(statusFieldErrorID).should("not.exist");
+		cy.get(tuitionFieldErrorID).should("not.exist");
+		cy.get(GroupFieldErrorID).should("not.exist");
+		cy.get(SubGroupFieldErrorID).should("not.exist");
+		cy.get(placeOfBirthFieldErrorID).should("not.exist");
+		cy.get(nationalityFieldErrorID).should("not.exist");
+		cy.get(firstLanguageFieldErrorID).should("not.exist");
+		cy.get(neighborhoodFieldErrorID).should("not.exist");
+		cy.get(addressFieldErrorID).should("not.exist");
+		cy.get(medicalAssuranceFieldErrorID).should("not.exist");
+		cy.get(emergencyFieldErrorID).should("not.exist");
+	});
+
+	it("shows error message on fields that have restrictions with an incorrect input", () => {
+		cy.get(basicInfoButtonID).click();
+		cy.get(editButtonID).click();
+
+		//check basic data
+		cy.testInput(CIFieldID, CIFieldErrorID, "", "50137758", "Se deben ingresar solo números, sin puntos ni guiones y no puede quedar vacía");
```

Comment:
```suggestion
		cy.testInput(CIFieldID, CIFieldErrorID, "", "50137758", "Se deben ingresar solo números y letras, sin puntos ni guiones y no puede quedar vacía");
```
el mensaje de error es este